### DISCUSSION TITLE
Blockquote attribution appears on a new page

### DISCUSSION THREAD

**Discussion by h0nzZik** [typst/typst#4244] [Q&A] [answered]
Suppose we have a block quote at the end of a page:
```typst
#lorem(590)
#quote(block: true, attribution: [Somebody])[#lorem(50)]
```
It happens for me that Typst puts the attribution at the beginning of a new page, as the first thing on the page. Is there a way of preventing that?

**Comment by Enivex** [comment, marked as answer]
Wrapping the quote in a block will work.

```
#lorem(590)
#block(quote(block: true, attribution: [Somebody])[#lorem(50)])
```

Though I'd consider the fact that you need to do that a bug. The orphan/widow-prevention doesn't always work correctly.

**Comment by laurmaedje** [comment]
In principle, `#show quote: set block(breakable: false)` should have worked, but it doesn't, because the attribution is for some reason not part of the internal block. That's a bug.
